Update pom.xml

akram opened the pull request.

DuncanDoyle commented:
I guess this is needed to build the project outside of BC, e.g. doing a normal Maven build from command line, or an S2I OpenShift build, right?

Did you test doing a build within business-central using this new POM? I seem to recall that adding that dependency gave errors in the BC build in the past?

Also, I think the scope of that dependency needs to be set to "provided". That thing is already on the classpath of the deployment in BC and KIE-Server IIRC.

akram commented:
Sure for the scope. Added there.

Still does not fix my issue with webservices, but at least having the kie-api is a requirement.

Sorry, I was commiting on my master branch. I just took your comment into account and squashed my two commits
